release-content/0.16/migration-guides/18083_Improved_DeriveEvent.md

```diff
@@ -0,0 +1,22 @@
+
+In Bevy 0.16 you can now use `#[derive(Event)]` for more specialized implementations. 
+
+```rust
+
+// before
+use bevy_ecs::prelude::*;
+
```

Comment:
```suggestion
```

I don't think specifying the import is necessary, but if you'd rather keep it we make all imports go through the `bevy` crate instead of sub-crates. (E.g. `use bevy::ecs::Event`)